central/scannerdefinitions/handler/handler.go

```diff
@@ -365,6 +370,39 @@ func (h *httpHandler) openMostRecentDefinitions(ctx context.Context, uuid string
 	return
 }
 
+func (h *httpHandler) openMostRecentMappings(fileName string) (file *vulDefFile, err error) {
```

Comment:
This is almost the same as `openMostRecentDefinitions`. I think we should try to avoid repeating all this code and abstract out the similarities into a separate function